core/src/test/java/com/linecorp/armeria/server/file/FileServiceBuilderTest.java

```diff
@@ -20,9 +20,45 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.linecorp.armeria.client.WebClient;
+import com.linecorp.armeria.common.AggregatedHttpResponse;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.common.annotation.Nullable;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.testing.junit5.server.ServerExtension;
 
 class FileServiceBuilderTest {
 
+    private static final String baseResourceDir =
+      FileServiceBuilderTest.class.getPackage().getName().replace('.', '/') + '/';
+
+    @RegisterExtension
+    static final ServerExtension server = new ServerExtension() {
+        @Override
+        protected void configure(ServerBuilder sb) {
+            sb.serviceUnder(
+              "/mimeTypeFunction",
+              FileService.builder(getClass().getClassLoader(), baseResourceDir + "baz")
+                .serveCompressedFiles(true)
+                .autoDecompress(true)
+                .maxCacheEntries(0)
+                .mimeTypeFunction(new MimeTypeFunction() {
+                    @Override
+                    public MediaType guessFromPath(String path) {
+                        return MediaType.JSON_UTF_8;
```

Comment:
I think this does not guarantee if the `mimeTypeFunction` works or not because it just returns `JSON_UTF_8`.
How about adding if conditions and also adding more test cases?
```java
public MediaType guessFromPath(String path) {
    if (path.endsWith(".json")) {
        return MediaType.JSON_UTF_8
    }
    if (path.endsWith(".txt")) {...}
    ...
}
```

Reply:
Agreed, I'll correct that. Thanks